string-manipulation-2/src/num-vowels.js

```diff
@@ -0,0 +1,36 @@
+/* exported numVowels */
+
+// dec num var counter
+// Enter loop
+// Create switch statement that finds vowels
+// case increments counter
+// return counter
+
+function numVowels(string) {
+  var counter = 0;
+  for (var i = 0; i < string.length; i++) {
+
+    switch (string[i].toLowerCase()) {
+      case 'a':
+        counter++;
+        break;
+      case 'e':
+        counter++;
+        break;
+      case 'i':
+        counter++;
+        break;
+      case 'o':
+        counter++;
+        break;
+      case 'u':
+        counter++;
+        break;
```

Comment:
You should make use of fall-through here

Reply:
I tried remove the "break;"  statements and the tests failed.

Reply:
It would be something like this:
```
    switch (string[i].toLowerCase()) {
      case 'a':
      case 'e':
      case 'i':
      case 'o':
      case 'u':
        counter++;
        break;
}
```